packages/suite/src/components/wallet/CoinjoinSummary/BalanceSection.tsx

```diff
@@ -39,29 +51,21 @@ export const BalanceSection = ({ account }: BalanceSectionProps) => {
     const goToSetup = () => goto('wallet-anonymize', { preserveParams: true });
 
     return (
-        <>
-            {/* temporary - content will be reworked and moved into the container below */}
-            {session && (
-                <Container>
-                    <CoinjoinSessionStatus account={account} session={session} />
-                </Container>
-            )}
+        <Container>
```

Comment:
not responsive enough
![image](https://user-images.githubusercontent.com/3729633/193056293-e2092b38-2c52-4891-aeb4-4a1db110d671.png)

Reply:
https://github.com/trezor/trezor-suite/issues/6473